### ISSUE TITLE
📃: ADD New Project Text Editor with Undo/Redo Functionality

### ISSUE THREAD

**Issue by Sumanth077s** [iamrahulmahato/master-web-development#1495]
🔴 Title : Text Editor with Undo/Redo Functionality
🔴 Tech stack : HTML,CSS,JS
🔴 Objective : Text Editor with Undo and Redo Functionality: A simple tool for managing text changes effortlessly.
🔴 Summary :N/A
📸 Screenshots: N/A

***********************************************************************

✅ Details to Include When Taking the Issue:
Name : Sumanth U
Participant Role (Specify the Open Source Program name, e.g., GSSOC, Hacktoberfest, etc.):Contributer at gssoc and hacktober fest

***********************************************************************

Happy Contributing! 🚀

Wishing you all the best on your open source journey. Enjoy! 😎

**Comment by pbln** [comment]
I want to work on this issue . 
Name : Prachi 
contributer at gssoc and hacktoberfest.

**Comment by Sumanth077s** [comment]
@pbln It's done

**Comment by Sumanth077s** [comment]
@pbln Raise a new issue and then start contributing

**Comment by pbln** [comment]
ok , actually , I am new to open source so can you pls give like an idea how can i contribute here ?

**Comment by Sumanth077s** [comment]
@pbln yeah sure, in project folder check for projects which are not in folder then raise a new issue about the project you want to contribute. Then add your project via pull request and mention Issue number in the pull request title and submit. The admin will review. If the project is unique he will merge it. Refer readme.md and contributing.md file for more info

**Comment by pbln** [comment]
@Sumanth077s  thanks a lot , this was really helpful!
